index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Brighton Times</title>
+    <title>icathyiBrighton Times</title>
```

Comment:
Hi, I believe there's no need for the first text.

Reply:
This is necessary for the demo.